infra/media/mediaInfoParser/KAMFMediaInfoParser.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+class KAMFData
+{
+    public $pts;
+    public $timestamp;
+
+};
+
+class KAMFMediaInfoParser extends KBaseMediaParser{
```

Comment:
KBaseMediaParser::getMediaInfo expected to return KalturaMediaInfo, so either return KalturaMediaInfo in this method and use a different method to return the AMFs, or, don't extend this class.